requirements.txt

```diff
@@ -8,12 +8,11 @@ aioredis
 aiozk==0.14.0
 async-timeout
 click==7.0                 # via Flask
-dash==1.0.2
-dash-core-components==1.0.0
-dash-html-components==1.0.0
-dash-table==4.0.2
-dash-dangerously-set-inner-html==0.0.2
-dash-renderer==1.0.0       # via dash
```

Comment:
Interesting that this vanishes - did it get absorbed into dash itself?

Reply:
yes, it's obsolete -- merged into the main dash repo